src/libtiled/isometricrenderer.cpp

```diff
@@ -491,7 +512,7 @@ void IsometricRenderer::drawMapObject(QPainter *painter,
             break;
         }
         case MapObject::Polyline: {
-            const QPointF &pos = object->position();
+            const QPointF &pos = bounds.topLeft();
```

Comment:
Why all the changing of this for `Point`, `Polygon` and `Polyline`? None of these objects have a size so it should be the same.

Reply:
For point, everything else used top-left, so I changed it as well for consistency.

I thought polygon and polyline had sizes, though. If they do not, it makes more sense for all three to revert to object->position. I'll make that change.

Reply:
Yeah, a polygon object only has an implicit size based its polygon, but things would get really weird if we'd start applying the alignment option based on that. The position of the polygon is already not the top-left as it is for rectangle objects, it's wherever you have placed the first point (it'd make sense to render this somehow).